TypeCobol.Test/Parser/Programs/Cobol85/GlobalNested.cbl

```diff
@@ -0,0 +1,43 @@
+IDENTIFICATION DIVISION.
+PROGRAM-ID. EnclosingPgm.
+DATA DIVISION.
+LOCAL-STORAGE SECTION.
+
+01 MyGlobalVar PIC X(5) GLOBAL.
+01 MyNotGlobalVar PIC X(5).
+
+01 MyGlobalGroup GLOBAL.
+  05 MGG1 PIC X(5).
+    10 MGG2 PIC X(5).
+
+01 MyNotGlobalGroup.
+  05 MNGG1 PIC X(5).
+    10 MNGG2 PIC X(5).
```

Comment:
Previous tests are ok, but can you add a test with a typed variable:
05 MNGGDate Type Date.